index.d.ts

```diff
@@ -5,6 +5,14 @@ declare const figureSet: {
 	readonly square: string;
 	readonly squareSmall: string;
 	readonly squareSmallFilled: string;
+	readonly squareLight: string;
+	readonly squareMedium: string;
+	readonly squareDark: string;
+	readonly squareTop: string;
+	readonly squareBottom: string;
+	readonly squareLeft: string;
+	readonly squareRight: string;
+	readonly squareCenter: string;
```

Comment:
`squareCenter` has margins on top, bottom but not on left and right. I am not sure whether this name conveys this properly.